src/docs/asciidoc/modules/ROOT/pages/chapter4.adoc

```diff
@@ -50,90 +50,162 @@ The consequences of architecture decisions, like adopting microservices, usually
 [quote, Martin Fowler]
 You shouldn't start with a microservices architecture. Instead, begin with a monolith, keep it modular, and split it into microservices once the monolith becomes a problem.
 
+== Reactive Java microservices
+
+Spring Boot 2.0 introduced a new web framework called Spring WebFlux. Previous versions of Spring Boot only shipped with Spring MVC as an option. WebFlux offers a way for developers to do _reactive programming_. This means you can write your code with familiar syntax and, as a result, your app will use fewer resources and scale better.
```

Comment:
I see you have reactive programming defined in the [actions section](https://github.com/mraible/jhipster-book/pull/76/files#diff-234a062e41ebd4f371065cfa2f51ea245a696e25531e5a00eaf4fc77ac72be17R27), I think it might will be helpful to the reader to include a hyperlink to the citation as reactive programming isn't defined in the text here.